detox/e2e/support/ui/screen/channel_info.js

```diff
@@ -2,34 +2,40 @@
 // See LICENSE.txt for license information.
 
 import {ChannelScreen} from '@support/ui/screen';
+import {timeouts, wait} from '@support/utils';
 
 class ChannelInfoScreen {
     testID = {
         channelInfoScreen: 'channel_info.screen',
-        channelInfoClose: 'screen.channel_info.close',
+        closeChannelInfoButton: 'close.channel_info.button',
         channelIconGMMemberCount: 'channel_icon.gm_member_count',
+        notificationPreferenceAction: 'channel_info.notification_preference.action',
+        editChannelAction: 'channel_info.edit_channel.action',
     }
 
     channelInfoScreen = element(by.id(this.testID.channelInfoScreen));
-    channelInfoClose = element(by.id(this.testID.channelInfoClose));
+    closeChannelInfoButton = element(by.id(this.testID.closeChannelInfoButton));
     channelIconGMMemberCount = element(by.id(this.testID.channelIconGMMemberCount));
+    notificationPreferenceAction = element(by.id(this.testID.notificationPreferenceAction));
+    editChannelAction = element(by.id(this.testID.editChannelAction));
 
     toBeVisible = async () => {
-        const screen = await this.channelInfoScreen;
-        await expect(screen).toBeVisible();
+        await wait(timeouts.TWO_SEC);
```

Comment:
why this long?

Reply:
I tried several values less than 2 seconds and more often they were not stable. 2 sec seem to be a magic number. There might be another underlying issue but I couldn't figure out why it wasn't stable with time less than 2 seconds. If you have suggestions on what to explore, let me know.